ui/shared/components/panel/search/constants.js

```diff
@@ -27,6 +27,7 @@ import {
 } from 'shared/utils/constants'
 
 import LocusListItemsFilter from './LocusListItemsFilter'
+import PaMoiSelector from '../../../../pages/Search/components/filters/PaMoiSelector'
```

Comment:
On principle, we never import from `pages` into `shared`, so you should move this component

Reply:
Ok I have moved `PaMoiSelector` from `pages/Search/components/filters/PaMoiSelector.jsx` to `ui/shared/components/panel/search/PaMoiSelector.jsx` 👍 

Is the "Selector" part of the name ok? I was originally just following the "LocusListSelector.jsx" name.

We have a bit more logic here than just a regular Dropdown, so `PaMoiSelector.jsx` suits the file better than `PaMoiDropdown.jsx`

I hope that's ok and I'm not using the word wrong. I think it fit this from the [Redux website](https://redux.js.org/usage/deriving-data-selectors#calculating-derived-data-with-selectors):

> 
> Calculating Derived Data with Selectors
> In a typical Redux application, the logic for deriving data is usually written as functions we call selectors.
> 
> Selectors are primarily used to encapsulate logic for looking up specific values from state, logic for actually deriving values, and improving performance by avoiding unnecessary recalculations.
>

Reply:
So in isolation `Dropdown` would probably still be a better name, but since it is used in conjunction with a component  named `LocusListSelector` I think this name is more consistent so we should keep it as is, but thanks for checking